scheduler-portal/src/main/java/org/ow2/proactive_grid_cloud_portal/scheduler/client/SubmitWindow.java

```diff
@@ -392,9 +392,26 @@ private void fillVarsPart(String jobDescriptor) {
     }
 
     private Widget prepareWorlflowInformationWidget() {
-        return new HTML("<center> <img src=" + icon + " height=40px width=40px> <br> <b>Documentation :</b> <a href=" +
-                        documentation + " target=_blank>" + documentation + " </a><br> <b> Bucket name :</b> " +
-                        bucketNameGI + " </center>");
+
+        String widgetIcon = "Please add an icon to the workflow <br>";
```

Comment:
Is it possible to add the icon from this interface? If not, I think showing nothing might be a better solution than this message

Reply:
I put "No icon"